frontend/test/metabase/scenarios/native/reproductions/21034-autocomplete-flicker.cy.spec.js

```diff
@@ -0,0 +1,28 @@
+import { restore, openNativeEditor } from "__support__/e2e/cypress";
+
+describe.skip("issue 21034", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    openNativeEditor();
+    cy.intercept(
+      "GET",
+      "/api/database/**/autocomplete_suggestions?**",
+      cy.spy().as("suggestions"),
+    );
+  });
+
+  it("should not invoke API calls for autocomplete twice in a row (metabase#18148)", () => {
+    cy.get(".ace_content")
+      .should("be.visible")
+      .type("p");
+
+    // Wait until another explicit autocomplete is triggered
+    // See https://github.com/metabase/metabase/pull/20970
+    cy.wait(1000);
```

Comment:
We shouldn't wait arbitrarily. Is this a debounce value or how did you arrive to 1000ms?

Reply:
Long enough to be more than `AUTOCOMPLETE_DEBOUNCE_DURATION` in `frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx`. Not sure how we should get that value (if anything).

Reply:
In theory, we can export that constant and import it in this test, then add a couple hundred milliseconds.
But the comment will suffice for now.

Reply:
Good idea, I've added a comment.